test/functional/wallet_multiwallet.py

```diff
@@ -129,15 +126,33 @@ def wallet_file(name):
         for wallet_name in to_load:
             self.nodes[0].loadwallet(wallet_name)
 
-        os.mkdir(wallet_dir('no_access'))
-        os.chmod(wallet_dir('no_access'), 0)
-        try:
+        # Tests for possible scanning errors:
+        # 0. Baseline, no errors.
+        with self.nodes[0].assert_debug_log(expected_msgs=[], unexpected_msgs=['Error scanning']):
+            walletlist = self.nodes[0].listwalletdir()['wallets']
+        assert_equal(sorted(map(lambda w: w['name'], walletlist)), sorted(in_wallet_dir))
+        # 1. "Permission denied" error.
+        if platform.system() != 'Windows' and os.geteuid() != 0:
```

Comment:
nit: Could add a warning when skipping the test because of user permissions:
```suggestion
        if platform.system() != 'Windows':
            if os.geteuid() == 0:
                self.log.warning('Skipping "permission denied"-test requiring non-root user.')
            else:
```

(Ran into something similar on my own branch as sanitizer-tests apparently run with root permissions ([CI log](https://github.com/hodlinator/bitcoin/actions/runs/13219450078/job/36902673995#step:7:5018))).

Reply:
Thanks! Applied.